stories/core/Surface.stories.tsx

```diff
@@ -0,0 +1,41 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+import { Story, Meta } from '@storybook/react';
+import { CreeveyMeta } from 'creevey';
+import React from 'react';
+import { Surface, SurfaceProps } from '../../src/core';
+
+export default {
+  component: Surface,
+  argTypes: {
+    className: { control: { disable: true } },
+    style: { control: { disable: true } },
+    elevation: {
+      options: [2, 4, 8, 16, 24],
+    },
+  },
+  parameters: {
+    creevey: {
+      captureElement: null,
+    },
+  },
+  title: 'Core/Surface',
+} as Meta<SurfaceProps> & CreeveyMeta;
+
+const cardStyle = {
```

Comment:
I added additional styling to the story to make it look similar to the tiles shown in the CSS demo

Before: 
![SurfaceBefore](https://user-images.githubusercontent.com/78564221/163051915-22d2e65b-5498-4ec0-b6ca-f98c57eb8683.PNG)

After:
![SurfaceAfter](https://user-images.githubusercontent.com/78564221/163052224-3139fb20-cb90-4de1-ae9c-5e48d12bcb94.PNG)

Reply:
Then I would put that inline so users could see code too..

Reply:
Code has been moved